tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot

```diff
@@ -142,6 +142,14 @@ Spawn Notebook With Arguments
       END
       Wait Until Element is Visible  id:progress-bar
       Run Keyword And Continue On Failure  Wait Until Page Does Not Contain Element  id:progress-bar  ${spawner_timeout}
+      Wait for JupyterLab Splash Screen  timeout=30
+      Maybe Close Popup
+      ${is_launcher_selected} =  Run Keyword And Return Status  JupyterLab Launcher Tab Is Selected
+      Run Keyword If  not ${is_launcher_selected}  Open JupyterLab Launcher
+      Launch a new JupyterLab Document  kernel=Python 3 (ipykernel)
```

Comment:
kernel=Python 3 (ipykernel) will be available for images with Elyra plugin like SDS,TF,PT. It will not be present for minimal,open vino etc which will break the script as the locator will not be found.

Reply:
You're right, I will need to find a way to work around this

Reply:
Oneway to do it will be create a list of images with and without Elyra plugin and the use IF condition to perform the respective task

Reply:
Yes, either that or I think I should be able to use `File -> New -> Notebook` which should work regardless of the image

Reply:
yes ...Either one of them should be good.Thnx